utils/deprecated.js

```diff
@@ -57,14 +30,3 @@ export function deprecated( ...params ) {
 
 	return originalDeprecated( ...params );
 }
-
-// viewport
-export function isExtraSmall() {
-	originalDeprecated( 'wp.utils.isExtraSmall', {
```

Comment:
Unfortunately the custom ESLint rule is not smart enough to capture anything except the default function name `deprecated` 😅

Reply:
Right, I was to creative renaming its name :)